core/tests/linter_tests/invalid_codeowners

```diff
@@ -0,0 +1,546 @@
+# Per-directory ownership and automatic assignment for pull requests.
```

Comment:
Can we make this a small file? [not sure!]

Reply:
Done.